core-web/libs/edit-content/src/lib/components/dot-edit-content-aside/dot-edit-content-aside.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="edit-content-aside--header">
+    <strong style="display: flex; align-items: center; gap: 0.3em">
+        <i class="pi pi-info-circle"></i>
+        {{ 'Information' | dm }}</strong
+    >
+    <dot-copy-button
+        *ngIf="contentLet?.inode"
+        [copy]="contentLet?.inode"
+        [attr.data-testId]="'inode'"
+        [label]="contentLet?.inode | slice : 0 : 8"></dot-copy-button>
+
+    <dot-api-link
+        *ngIf="contentLet?.stInode"
+        [href]="'api/v1/contenttype/id/' + contentLet?.stInode"></dot-api-link>
+</div>
+
+<dl class="edit-content-aside--item">
+    <dt>{{ 'Content State' | dm }}</dt>
+
+    <p-chip
+        *ngIf="contentLet"
+        [styleClass]="(contentLet | contentletStatus).classes"
+        [label]="(contentLet | contentletStatus).label"></p-chip>
+</dl>
+
+<dl class="edit-content-aside--item">
+    <dt>{{ 'Modified by' | dm }}</dt>
+    <dd class="font-bold" *ngIf="contentLet" data-testId="modified-by">
+        {{ contentLet?.modUserName }}
+    </dd>
+</dl>
+
+<dl class="edit-content-aside--item">
+    <dt>{{ 'Last modified' | dm }}</dt>
+    <dd class="font-bold" *ngIf="contentLet" data-testId="last-modified">
+        {{ contentLet?.modDate | dotRelativeDate }}
+    </dd>
+</dl>
+
+<dl class="edit-content-aside--item">
+    <dt>{{ 'Content Type' | dm }}</dt>
+    <a [href]="'#/content-types-angular/edit/' + contentType" data-testId="content-type">{{
+        contentType
+    }}</a>
+</dl>
```

Comment:
CSS grid is two dimensional, and you can arrange layouts in columns and rows, so you don't need (and should not have) a `<dl>` per row... the whole idea is to have one `<dl>` to semantically group this content.

```html
<dl class="edit-content-aside">
    <dt>Modified by</dt>
    <dd>[ModUserName]</dd>

    <dt>Last modified</dt>
    <dd[ModDate]</dd>

    <dt>Content Type</dt>
    <dd><a href="#/content-types-angular/edit/[contentType]" data-testId="content-type">[ContentType]</a></dd>
</dl>
```

```css
    .edit-content-aside {
      display: grid;
      grid-template-columns: 1fr 1fr;
      gap: 10px;
    }
```

Reply:
with this approach, when we dont have a value (like status, last modified), we also render a dd but empty, this is correct?
In otherwise, we dont render the dd, so in that case we need to render a two dimensional grid (if the dd is not render, the next dt is render beside)

With data (On existing contenlet)
![Screenshot 2023-12-08 at 15 19 36](https://github.com/dotCMS/core/assets/144152756/91c68c91-ca98-4f46-8636-1aa1c95c4e38)

Without data (On new contentlet)
![Screenshot 2023-12-08 at 15 19 42](https://github.com/dotCMS/core/assets/144152756/d8c029d1-c36c-4e2b-a833-962c56710abd)